### PR TITLE
adding LVK mission page

### PR DESCRIPTION
**Pull request by jracusin**

### CODE REVIEW

File: app/routes/missions/lvk.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+/>
+
+**Planned Lifetime:** 2028+
+
+**End of Operations:** No specific requirement
+
+**Data Archive:**
+https://www.gw-openscience.org
+
+LIGO, Virgo, and KAGRA comprise the advanced gravitational wave network. LIGO is operated by the National Science Foundation, Virgo by the European Gravitational Wave Observatory, and KAGRA by the ministry of Education, Culture, Sports, Science and Technology-Japan (MEXT). Together they detect, localize, and characterize the coalescence of compact binary mergers, continuous gravitational waves, and burst gravitational waves.
```
Comment (lpsinger) [inline app/routes/missions/lvk.md:49]:
```suggestion
LIGO, Virgo, and KAGRA comprise the advanced gravitational wave detector network. LIGO is operated by the National Science Foundation, Virgo by the European Gravitational Wave Observatory, and KAGRA by the ministry of Education, Culture, Sports, Science and Technology-Japan (MEXT). Together they detect, localize, and characterize the coalescence of compact binary mergers, continuous gravitational waves, and burst gravitational waves.
```

File: app/routes/missions/lvk.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+**GCN Notice Types in GCN Classic and GCN Classic Over Kafka:**
+[Detailed Descriptions and Examples](https://gcn.gsfc.nasa.gov/lvc.html)
+
+| Type              | Contents                                                                                      | Latency    |
+| ----------------- | --------------------------------------------------------------------------------------------- | ---------- |
+| `LVC_PRELIMINARY` | 1st Notice, Timestamp Alert, location probability skymap; all automated processing.           | 1-10 min   |
+| `LVC_INITIAL`     | Improved SkyMap now available (human-involved procesing).                                     | 4-24 hr    |
+| `LVC_UPDATE`      | Ultimate refined skymap (more detailed off-line processing).                                  | ~1-7 days  |
+| `LVC_RETRACTION`  | After human analysis/evaluation, a retraction will be issued if trigger is not astrophysical. | 1hr - 1day |
```
Comment (lpsinger) [inline app/routes/missions/lvk.md:59]: Several changes here:

1. Rewrite "1st" as a word.
2. Use en dashes for numerical ranges (option + dash on macOS keyboards)
3. Use SI unit abbreviations
4. Replace "skymap" with "sky map"; "skymap" is not a word.

```suggestion
| `LVC_PRELIMINARY` | First Notice, Timestamp Alert, location probability sky map; all automated processing.           | 1–10 min   |
| `LVC_INITIAL`     | Improved SkyMap now available (human-involved procesing).                                     | 4–24 h    |
| `LVC_UPDATE`      | Ultimate refined skymap (more detailed off-line processing).                                  | ~1–7 d  |
| `LVC_RETRACTION`  | After human analysis/evaluation, a retraction will be issued if trigger is not astrophysical. | 1 h–1 d |
```

**Review by lpsinger — CHANGES_REQUESTED** [review]
Find SVG graphics for EGO + Virgo and LIGO if possible.

**Comment by jracusin** [comment]
> Find SVG graphics for EGO + Virgo and LIGO if possible.

I actually tried to find sag versions and could not.

**Review by lpsinger — APPROVED** [review]

**Pull request merged**